src/App.jsx

```diff
@@ -14,70 +15,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setValue] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood === '' ? (
+          'No goods selected'
+        ) : (
+          `${selectedGood} is selected`
+        )}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+        {selectedGood !== '' && (
```

Comment:
Use logical operators 
```suggestion
        {!!selectedGood && (
```